symplyphysics/laws/electricity/waveguides/maximum_electric_field_intensity_in_waveguide.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Eq, solve, pi, sqrt
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless)
+
+## Description
+## A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
```

Comment:
This law is about rectangular waveguides? Add `rectangular` to file name. Mention it in conditions.